indexer-app/src/cmr/indexer/data/concepts/collection/opendata.clj

```diff
@@ -32,11 +32,15 @@
   "Returns the opendata related url for the given collection related url"
   [related-url]
   (let [{:keys [Description Type Subtype URL]} related-url
+        Format (get-in related-url [:GetData :Format])
+        get-data-mime-type (get-in related-url [:GetData :MimeType])
         MimeType (get-in related-url [:GetService :MimeType])
         size (get-in related-url [:GetData :Size])]
     {:type Type
      :sub-type Subtype
      :url URL
+     :format Format
+     :get-data-mime-type get-data-mime-type
      :description Description
      :mime-type MimeType
```

Comment:
Since we added another mime-type to the map we should probably change this key to `:get-service-mime-type`.